lang/cs/Org.Apache.REEF.Network.Tests/GroupCommunication/GroupCommunicationTests.cs

```diff
@@ -380,6 +380,45 @@ public void TestBroadcastOperator()
             Assert.Equal(value, receiver2.Receive());
         }
 
+        /// <summary>
+        /// Test IBroadcastReceiver.Receive() with and without cancellation token
+        /// </summary>
+        [Fact]
+        public void TestBroadcastOperatorWithCancelation()
+        {
+            string groupName = "group1";
+            string operatorName = "broadcast";
+            string masterTaskId = "task0";
+            string driverId = "Driver Id";
+            int numTasks = 10;
+            int value = 1337;
+            int fanOut = 3;
```

Comment:
GroupCommDriver constructor requires it for all typologies. Eventually only used if the topology is tree. We can modify it later if we want.